addons/sale/report/sale_report.py

```diff
@@ -179,7 +179,7 @@ def _from_sale(self):
             """.format(
             currency_table=self.env['res.currency']._get_query_currency_table(
                 {
-                    'multi_company': True,
+                    'companies': [{'name': company.name, 'id': company.id} for company in self.env.companies],
```

Comment:
AFAICS, you only use the company id in `_get_query_currency_table`, so why do you add the name in the options dict ?
```suggestion
                    'companies': [{'id': company.id} for company in self.env.companies],
```

Reply:
Good point, thanks. I just put the the same (minimal) information that is used for report options in account_reports. I will wait until oco chimes in, but the name is not necessary here.